lib/fortnox/api/repositories/base/loaders.rb

```diff
@@ -27,19 +27,19 @@ def find( id_or_hash )
           id = Integer( id_or_hash )
           find_one_by( id )
 
-        catch ArgumentError
+        rescue ArgumentError
           raise ArgumentError, "find only accepts a number or hash as argument"
         end
 
         def find_one_by( id )
-          response_hash = get( @uri + id )
-          entity_hash = response_hash[ @json_entity_wrapper ]
+          response_hash = get( @options.uri + id.to_s )
```

Comment:
Use interpolation instead of manual casting and concatentation: `response_hash = get( "#{ @options.uri }#{ id }" )`